applications/StructuralMechanicsApplication/tests/formfinding_test/Fofi_4Point_Tent_noCable_test_parameters.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
Can you wait to merge this PR until the #370 is approved?, some minor changes were added to the tests, and maybe it is a good idea to merge after check that it is still working